.devcontainer/devcontainer.json

```diff
@@ -38,6 +47,50 @@
       "!include_dir_list scalar",
       "!include_dir_merge_list scalar",
       "!include_dir_merge_named scalar"
+    ],
+    // python common
+    "[python]": {
+      "editor.defaultFormatter": "ms-python.python",
+      "editor.codeActionsOnSave": {
+        "source.organizeImports": true
+      }
+    },
+    "python.linting.enabled": true,
+    "python.pythonPath": "/usr/local/bin/python",
+    // python linting
+    "python.linting.banditEnabled": true,
+    "python.linting.banditArgs": [
+      "--format=custom",
+      "--configfile=tests/bandit.yaml"
+    ],
+    "python.linting.flake8Enabled": true,
+    "python.linting.flake8Args": ["--config=./setup.cfg"],
+    "python.linting.mypyEnabled": true,
+    "python.linting.mypyArgs": [
+      "--follow-imports=silent",
+      "--ignore-missing-imports",
+      "--show-column-numbers",
+      "--no-pretty",
+      "--config-file=./mypy.ini"
+    ],
+    "python.linting.pylintEnabled": false,
+    "python.linting.pylamaEnabled": false,
+    "python.formatting.provider": "black",
+    "python.formatting.blackArgs": [
+      "--config=./pyproject.toml",
+      "--line-length=88"
+    ],
+    "isort.args": ["--settings-path=./pyproject.toml"],
+    "pylint.args": ["--rcfile=./pyproject.toml"],
+    // python tests
+    "python.testing.promptToConfigure": false,
+    // disabled by default due to extra needed requirements
+    "python.testing.pytestEnabled": false,
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestArgs": [
+      "--color=no"
```

Comment:
Personal preference? Should not be in here IMHO

Reply:
Old outstanding bug in VS Code Python. It does not support color output. https://github.com/microsoft/vscode-python/issues/12615

This goes with the other ones, explicitly defining color=no here makes sure the default is no, even if it is in the pytest config so it does not try render color escape characters.